src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/system_request.cc

```diff
@@ -511,6 +510,13 @@ void SystemRequest::Run() {
     return;
   }
 
+  if (mobile_apis::RequestType::QUERY_APPS == request_type &&
+      application_manager_.get_settings().max_supported_protocol_version() <
+          4) {
```

Comment:
@Ypostolov 4 is like a magic number extract ti variable with comment

Reply:
@LuxoftAKutsan 
Fixed in 3540848c21d445e1c079e44df650449f6b506029